packages/@vue/cli-service/lib/webpack/CorsPlugin.js

```diff
@@ -38,7 +38,7 @@ module.exports = class CorsPlugin {
               if (hash) {
                 tag.attributes.integrity = hash
               }
-            } else if (tag.tagName === 'link' && tag.attributes.rel === 'stylesheet') {
+            } else if (tag.tagName === 'link' && (tag.attributes.rel === 'stylesheet' || tag.attributes.rel === 'prefetch')) {
```

Comment:
If I'm not mistaken, the linked issue only said it's implemented for `preload` links, not `prefetch`.

And Line#49-56 needs to be removed too.